public/components/IndexMapping/IndexMapping.tsx

```diff
@@ -282,6 +295,30 @@ const IndexMapping = (
                     properties: [...(oldValue?.properties || []), ...transformObjectToArray(result?.properties || {})],
                   });
                 }}
+                path={`index-settings-${Date.now()}.json`}
+                diagnosticsOptions={{
+                  validate: true,
+                  schemas: [
+                    {
+                      fileMatch: ["index-settings-*.json"],
+                      schema: {
+                        title: "Product",
+                        description: "A product in the catalog",
```

Comment:
what does title and description used for?

Reply:
Fix that, this title and description will be displayed when we hover the most outside bracket.
![image](https://user-images.githubusercontent.com/13493605/230311051-2e488822-1d7f-4518-97b6-b165435344b0.png)